cmd/osde2ectl/get/cmd.go

```diff
@@ -65,10 +78,32 @@ func init() {
 		false,
 		"A flag that triggers the fetching of a given cluster's kubeconfig.",
 	)
+	pfs.Uint64VarP(
+		&args.hours,
+		"hours",
+		"r",
+		0,
+		"Cluster expiration extension value in hours.",
+	)
+
+	pfs.Uint64VarP(
+		&args.minutes,
+		"minutes",
+		"m",
+		0,
+		"Cluster expiration extension value in minutes.",
+	)
+
+	pfs.Uint64VarP(
```

Comment:
I don't think we need second-level resolution. This should always just default to zero.